app/queries/updated_archival_resources.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+class UpdatedArchivalResources
+  def self.queries
+    [:updated_archival_resources]
+  end
+
+  attr_reader :query_service
+  delegate :resource_factory, to: :query_service
+  delegate :orm_class, to: :resource_factory
+  delegate :run_query, to: :query_service
+  def initialize(query_service:)
+    @query_service = query_service
+  end
+
+  def updated_archival_resources(since_date:)
+    run_query(query(since_date))
+  end
+
+  def query(since_date)
+    <<-SQL
+      select * FROM orm_resources
+      WHERE metadata->>'archival_collection_code' is not null AND updated_at > '#{since_date}'
```

Comment:
Can you change this to be `WHERE metadata @> '{"archival_collection_code": []}' AND updated_at > ?`, then change the above to be `run_query(query, since_date.to_s)`?

`metadata->>'archival_collection_code'` doesn't use an index (unless you explicitly create one for that field), and we should use parameterized queries when we can just to be safe.

Reply:
Forgot to request changes.

Reply:
For reference, just ran these two queries on prod and it's a difference of 35s vs 7ms.

Reply:
👍 Just updated the query